install/kubernetes/cilium/charts/config/values.yaml

```diff
@@ -7,3 +7,6 @@
 # - azure
 # - crd
 ipam: "cluster-pool"
+#
+# kubeProxyReplacement healthz server bind address
+kubeProxyReplacementHealthzBindAddr: "0.0.0.0:10256"
```

Comment:
the lint is failing, this change requires a `make -C install/kubernetes`

Reply:
Done